packages/ast-spec/src/declaration/VariableDeclaration/spec.ts

```diff
@@ -20,16 +21,25 @@ export interface LetOrConstOrVarDeclaration extends BaseNode {
   declarations: LetOrConstOrVarDeclarator[];
   /**
    * Whether the declaration is `declare`d
-   * ```
+   * @example
+   * ```ts
    * declare const x = 1;
    * ```
    */
   declare: boolean;
   /**
    * The keyword used to declare the variable(s)
    * ```
+   * @example
```

Comment:
Same here: not sure why we want to split them up (and I think there's a stray `` ``` `` above)